events/wc_druidic_events.txt

```diff
@@ -115,9 +146,33 @@ character_event = {
 narrative_event = {
 	id = WCDRU.300
```

Comment:
Ok, now there's problem: the event chain doesn't stop if `event_target:target_mage` lost the artifacts.

Reply:
> There's a problem: the event chain doesn't stop if `event_target:target_mage` lost the artifacts. Logically, you can't summon the worgens after that.

The book isn't magical, it just contains specific information on the nature of the worgen and where to find them. Humans didn't know about the Emerald Nightmare until they befriended the night elves in Kalimdor (https://wow.gamepedia.com/Night_elf#Rediscovery_and_the_Third_War), so Ur was possibly the only human to have encountered it before then.

The target mage has presumably memorized the specifics needed to interact with the Emerald Dream when first studying the book, so loosing it shouldn't have any effect. The target mage is likely unlanded as well, so loosing it would mean the character is probably dead.

Reply:
>The target mage has presumably memorized the specifics needed to interact with the Emerald Dream when first studying the book

Target mage can be dull, slow or even imbecile, since nothing forbids it. I don't think it's possible for all characters to memorize specifics when first studying the book.

>The target mage is likely unlanded as well, so loosing it would mean the character is probably dead.

When you loot province, you can steal artifacts from unlanded characters without killing them